spec.html

```diff
@@ -26620,6 +26620,28 @@ <h1>Abstract Module Records</h1>
             </tr>
           </table>
         </emu-table>
+
+        <emu-clause id="sec-EvaluateModuleSync" type="abstract operation">
+          <h1>
+            EvaluateModuleSync (
+              _module_: a Module Record,
+            ): either a normal completion containing ~unused~ or a throw completion
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It synchronously evaluates _module_, provided that the caller guarantees that _module_'s evaluation will return an already settled promise.</dd>
+          </dl>
+
+          <emu-alg>
+            1. Assert: _module_ is not a Cyclic Module Record.
+            1. Let _promise_ be ! _module_.Evaluate().
+            1. Assert: _promise_.[[PromiseState]] is either ~fulfilled~ or ~rejected~.
+            1. If _promise_.[[PromiseState]] is ~rejected~, then
+              1. Perform HostPromiseRejectionTracker(_promise_, "handle").
+              1. Return ThrowCompletion(_promise_.[[PromiseResult]]).
+            1. Return NormalCompletion(~unused~).
```

Comment:
We don't need to explicitly wrap completions since the AO signature declares it returns a completion record.

```suggestion
            1. Return ~unused~.
```

Reply:
Strictly speaking [that rule](https://tc39.es/ecma262/multipage/notational-conventions.html#sec-implicit-normal-completion) only applies to algorithms and built-in functions, not to AOs.

(Unless you want to consider an AO to be "an algorithm", which... I mean I guess it is but that's not how I'd interpreted this paragraph.)

Reply:
Well that linked description starts with
> In algorithms within abstract operations which are declared to return a Completion Record, ...